src/types/module.rs

```diff
@@ -93,16 +93,53 @@ impl PyModule {
     ///
     /// # Examples
     ///
+    /// Include a file at compile time by using [`std::include_str` macro][1]:
+    ///
+    /// ```no_run
+    /// use pyo3::prelude::*;
+    ///
+    /// # fn main() {
+    ///       let py_foo = include_str!("../python/utils/foo.py");
+    ///       let py_app = include_str!("../python/app.py");
+    ///       let from_python = Python::with_gil(|py| -> PyResult<Py<PyAny>> {
+    ///           PyModule::from_code(py, py_foo, "utils.foo", "utils.foo")?;
+    ///           let app: Py<PyAny> = PyModule::from_code(py, py_app, "", "")?
+    ///               .getattr("run")?
+    ///               .into();
+    ///           app.call0(py)
+    ///       });
+    ///
+    ///       println!("py: {}", from_python?);
+    ///       Ok(())
```

Comment:
Given that the page is pretty long already, it can be a bit shorter and simpler:

```suggestion
    ///       let code = include_str!("../example.py");
    ///
    ///       Python::with_gil(|py| {
    ///            let module = PyModule::from_code(py, code, "example", "example")?;
    ///            Ok(())
    ///       })?;
    ///       # Ok(())
```

Reply:
changed in https://github.com/PyO3/pyo3/commit/467b1be4e54e965c889cb70d15dd65c30d721726 and made sure it compiles